.gitignore

```diff
@@ -40,6 +40,7 @@
 *.rmeta
 *.s
 *.so
+*.dylib
```

Comment:
Note that the list appears to be sorted.

Reply:
Sorted it after usecase. So so and dylib is basically the same

Reply:
Yes, but the rest of the list is not sorted by use case, and there are instructions in the top that say "(sorted alphabetically)".